### PR TITLE
Use pip 9

### PR DESCRIPTION
**Pull request by Kami**
Latest version (10.0.0) might break some things so we should use 9.0.0.1 for now to be on the safe side.

### CODE REVIEW

**Review by bigmstone — APPROVED** [review]
:+1:

File: actions/setup_e2e_tests.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ st2ctl reload --register-all
 
 # Robotframework requirements
 cd st2tests
-sudo pip install --upgrade pip
+sudo pip install --upgrade "pip>=9.0,<9.1"
 sudo pip install --upgrade virtualenv
```
Comment (arm4b) [inline actions/setup_e2e_tests.sh]: Need to pin virtualenv as well here (to `15.1.0` which ships with pip 9.0)

Comment (arm4b) [reply actions/setup_e2e_tests.sh]: And same thing, do `virtualenv --no-download` 2 lines below as well, otherwise it will be using latest pip 10.

Comment (Kami) [reply actions/setup_e2e_tests.sh]: Good catch - pushed those changes.

**Review by arm4b — APPROVED** [review]
:+1: 

Also a small code comment why we're doing it would be nice, so someone else looking at these lines in future and thinking about upgrading will have a better idea.

**Pull request merged**